Dockerfile

```diff
@@ -1,14 +1,11 @@
-FROM pudo/aleph-base:1.4
-MAINTAINER Friedrich Lindenberg <friedrich@pudo.org>
-ENV DEBIAN_FRONTEND noninteractive
-
-# Begin python festivities
-COPY requirements.txt /tmp/requirements.txt
-RUN pip install -q --upgrade pip && pip install -q functools32 \
-  && pip install -q -r /tmp/requirements.txt
+FROM code4sa/aleph-python-base
 
 COPY . /aleph
 WORKDIR /aleph
 ENV ALEPH_SETTINGS /aleph/contrib/docker_settings.py
-RUN pip install -q -e /aleph
-RUN rm -rf /aleph/.git && bower --allow-root --quiet install
+RUN pip install git+git://github.com/Code4SA/aleph_zagazettecrawler.git && \
+    pip install -q -e /aleph && \
+    rm -rf /aleph/.git && \
+    bower --allow-root --quiet install
+
+CMD newrelic-admin run-program gunicorn --workers 1 -b 0.0.0.0:5000 --worker-class gevent --timeout 600 --max-requests 3000 --max-requests-jitter 100 --log-file - --access-logfile - aleph.manage:app
```

Comment:
It shouldn't be necessary to tell gunicorn what port or IP to bind to. This caused me problems last time, because dokku thought it bound to another port. I think we should let it use the defaults.

Here's the line from our old dokku:

``CMD newrelic-admin run-program gunicorn --workers 1 --worker-class gevent --timeout 600 --max-requests 3000 --max-requests-jitter 100 --log-file - --access-logfile - aleph.manage:app``